app/presenters/finder_presenter.rb

```diff
@@ -55,6 +55,10 @@ def filters
     facets.filters
   end
 
+  def feed_disabled?
```

Comment:
What about calling it `feed_enabled?` and defaulting it to `true`? Avoiding double negatives is generally desirable